packages/react-components/src/util/toAddress.ts

```diff
@@ -12,9 +13,13 @@ export default function toAddress (value?: string | Uint8Array | null, allowIndi
         ? hexToU8a(value)
         : keyring.decodeAddress(value);
 
-      assert(allowIndices || u8a.length === 32, 'AccountIndex values not allowed');
+      assert(allowIndices || u8a.length === 32 || u8a.length === 20, 'AccountIndex values not allowed');
```

Comment:
This is one of the new changes after my original PR.

Reply:
Where can I find those changes? Should I just rebase to master?

Reply:
Yes. Would just rebase on master. This may be the only === 32 Check here, but a search for 32 should throw out the rest. 

Previous to this we never actually checked the sizes, however doing so does create some issues in chains where the config may be different. (Hence being explicit)